packages/node/test/integration/cross-node-multisig-creation.spec.ts

```diff
@@ -0,0 +1,57 @@
+import dotenv from "dotenv";
+import FirebaseServer from "firebase-server";
+
+import { IMessagingService, IStoreService, Node, NodeConfig } from "../../src";
+
+import { A_PRIVATE_KEY, B_PRIVATE_KEY } from "../env";
+
+import FirebaseServiceFactory from "./services/firebase-service";
+import { getChannelAddresses, getNewMultisig } from "./utils";
```

Comment:
AFAICT there are two different things named `getChannelAddresses`; can we disambiguate them

Reply:
What are the two different things?

Are you referring to these two:
- https://github.com/counterfactual/monorepo/pull/329/files#diff-aee0f6276c67926a5915f7fb7f49a8dcR53
- https://github.com/counterfactual/monorepo/pull/329/files#diff-888623f31f995189e8abe5b3979358e8R33

Reply:
yes, the wrapper in `utils`

actually, that wrapper is super thin and does "the expected thing", so maybe the same name is fine. in that case a short docstring in `utils` would help